app/src/libs/combat/tags.ts

```
@@ -11,6 +11,52 @@ import type { GeneralType } from "@/drizzle/constants";
+  target?: BattleUserState | undefined;
+  level: number | undefined;
+  round?: number;
+  barrierAbsorb?: number;
+  battle?: Battle; // Make battle optional since it's not always needed
+}): T => {
+  const { tag, user, target, level, round, barrierAbsorb, battle } = props;
+  if ("rounds" in tag) {
+    tag.timeTracker = {};
+  }
+  if ("power" in tag) {
+    tag.power = tag.power;
```

Biome (1.9.4)

[error] power is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)